### DISCUSSION TITLE
Logarithmic scale and color options directly on the charts page

### DISCUSSION THREAD

**Discussion by Drosaca** [home-assistant/frontend#18494] [Dashboard] [answered]
I would appreciate having the ability to access options right on the chart interface. For instance, having a checkbox located beneath the graph to enable the logarithmic scale display would be quite beneficial for rapidly growing data. Additionally, on the energy page, it would be convenient to be able to select graph colors simply by clicking on the colors in the legend. I've noticed that color customization is planned for the next update, but it appears that this feature can only be configured through the home assistant's configuration file.

**Comment by ildar170975** [comment, marked as answer]
> having a checkbox located beneath the graph to enable the logarithmic scale display would be quite beneficial for rapidly growing data

https://github.com/home-assistant/frontend/pull/18637
